pages/index.js

```diff
@@ -71,6 +71,9 @@ export default class Index extends React.Component {
             </div>
             <h1>Ethical<br />development<br />of keen web-apps</h1>
             <h2>You have a keen idea<span className="dash" />we have a keen approach</h2>
+            <Link href="/contacts?activeContactForm=false">
```

Comment:
In this case it's right to use a different approach. Contact form is active by default. We can use URL parameter for switch to second tab. I think this is more expected behavior